staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -445,6 +445,7 @@ message PodIP{
     string ip = 1;
 }
 // PodSandboxNetworkStatus is the status of the network for a PodSandbox.
+// Currently ignored for pods sharing the host networking namespace.
```

Comment:
Could you also update the v1alpha2 proto? We are keeping both files the same till we remove that file.